config_converter/config_mapper/config_mapper.py

```diff
@@ -79,8 +80,9 @@ def extract_root_dirs(config_obj: config_pb2.Directive) -> tuple:
         if d.name not in plugin_prefix_map:
             stats['entities_skipped'] += 1
             stats['attributes_skipped'] += _get_aggregated_num_attributes(d)
-            logging.warning('Currently we don\'t support plugins of %s',
-                            d.name)
+            logging.warning(
+                'Skipping mapping %s into master agent config file as its %s',
```

Comment:
Let's not use the `%s` for the 2nd part of the string just to meet the line limit. Instead, break a long line by using parentheses for implicit line joining. An example from https://google.github.io/styleguide/pyguide.html#32-line-length : 
('This will build a very long long '
     'long long long long long long string')

The 1st `%s` is fine as it is a dynamic variable depending on `d.name`.